nft/nfts.yaml

```diff
@@ -1093,6 +1095,12 @@ components:
         type: integer
         default: 100
       in: query
+    getOwnersForTokenPageSize:
+      name: pageSize
+      description: 'Number of owners to be returned per page.'
+      schema:
+        type: integer
+      in: query
     contractAddresses:
```

Comment:
@joshzhang5 can u add the default?

Reply:
The default is 1 billion rn heh because this endpoint didn't use to be paginated and we don't want to break behavior, the plan is to make a breaking change later but don't want people depending on this billion number!

Reply:
ah got it lol